packages/nakama-js/socket.ts

```diff
@@ -765,6 +767,12 @@ export class DefaultSocket implements Socket {
 
     this.adapter.onMessage = (message: any) => {
       if (this.verbose && window && window.console) {
+        if (message.match_data && message.match_data.data) {
+          message.match_data.data = decode(message.match_data.data);
```

Comment:
isn't this decoding the match data twice now? once in the adapter and now once here?

Reply:
Removed